src/authorities/OverdueInstanceAuthority.ts

```diff
@@ -0,0 +1,52 @@
+import {inject, injectable} from 'inversify';
+import PS2AlertsInstanceInterface from '../interfaces/PS2AlertsInstanceInterface';
+import {getLogger} from '../logger';
+import {TYPES} from '../constants/types';
+import InstanceHandlerInterface from '../interfaces/InstanceHandlerInterface';
+import {filter} from 'lodash';
+
+@injectable()
+export default class OverdueInstanceAuthority {
+    private static readonly logger = getLogger('OverdueInstanceAuthority');
+
+    private readonly instanceHandler: InstanceHandlerInterface;
+
+    private timer: NodeJS.Timeout | null = null;
+
+    constructor(@inject(TYPES.instanceHandlerInterface) instanceHandler: InstanceHandlerInterface) {
+        this.instanceHandler = instanceHandler;
+    }
+
+    public run(): void {
+        OverdueInstanceAuthority.logger.debug('Created OverdueInstanceAuthority timer');
+
+        if (this.timer) {
+            OverdueInstanceAuthority.logger.error('Attempted to run OverdueInstanceAuthority timer when already defined!');
+            clearInterval(this.timer);
+        }
+
+        this.timer = setInterval(() => {
+            OverdueInstanceAuthority.logger.debug('Running OverdueInstanceAuthority overdue alert check');
+
+            filter(this.instanceHandler.getAllInstances(), (instance) => {
```

Comment:
Why use filter when you can use filter :D

`src/handlers/aggregate/instance/InstancePopulationAggregate.ts` line something